f5_openstack_agent/lbaasv2/drivers/bigip/agent.py

```diff
@@ -33,10 +33,10 @@
 
 
 try:
-    from neutron.agent.common import config
     from neutron.agent.linux import interface
     from neutron.common import config as common_config
     from neutron.common import rpc as n_rpc
+    from neutron.conf.agent import common as config
```

Comment:
I'm wondering if this code change will run in Ocata and Newton. We should test this change in Ocata and Newton before we commit, if you haven't already. You might need to put the import into a try/except block.